<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RL-A2000-Test</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="TypeSelctcomboBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <item>
     <property name="text">
      <string>L100</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="Connect_Btn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
   <widget class="QComboBox" name="COMSelectcomboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
      <horstretch>20</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Quit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>710</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;华文楷体&quot;;</string>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PUMP_WASH">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>560</y>
      <width>251</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>开始清洗</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>1231</width>
      <height>471</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="Set_Injuct_M">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>231</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文楷体</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>注液量M</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Injuct_Count_W2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>210</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>50</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Set_Injuct_W3">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>190</y>
       <width>231</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文楷体</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>注液量W3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Set_Injuct_R2">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>20</y>
       <width>231</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文楷体</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>注液量R2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Set_Injuct_W4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>280</y>
       <width>231</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文楷体</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>注液量W4</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Injuct_Count_W3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>210</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>50</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Set_Injuct_W1">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>100</y>
       <width>231</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文楷体</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>注液量W1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Injuct_Count_W4">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>300</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>50</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Injuct_Count_R2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>30</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>24</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>300</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Set_Injuct_W2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>190</y>
       <width>231</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文楷体</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>注液量W2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Injuct_Count_W1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>120</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>300</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Injuct_Count_M">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>120</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>300</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Injuct_Count_R1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>24</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>300</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Set_Injuct_R1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>231</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文楷体</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>注液量R1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Set_Injuct_Base">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>370</y>
       <width>231</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文楷体</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>注液量底物</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Injuct_Count_Base">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>390</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>50</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Set_Injuct_W5">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>280</y>
       <width>231</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文楷体</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>注液量W5</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Injuct_Count_W5">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>300</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>50</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Set_Injuct_W6">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>370</y>
       <width>231</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文楷体</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>注液量W6</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Injuct_Count_W6">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>390</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>50</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="L100_Filling">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>590</y>
      <width>331</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>开始灌装</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>1</number>
    </property>
    <property name="autoRepeatInterval">
     <number>1</number>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>TypeSelctcomboBox</tabstop>
  <tabstop>Connect_Btn</tabstop>
  <tabstop>COMSelectcomboBox</tabstop>
  <tabstop>Quit</tabstop>
  <tabstop>PUMP_WASH</tabstop>
  <tabstop>Set_Injuct_M</tabstop>
  <tabstop>Injuct_Count_W2</tabstop>
  <tabstop>Set_Injuct_W3</tabstop>
  <tabstop>Set_Injuct_R2</tabstop>
  <tabstop>Set_Injuct_W4</tabstop>
  <tabstop>Injuct_Count_W3</tabstop>
  <tabstop>Set_Injuct_W1</tabstop>
  <tabstop>Injuct_Count_W4</tabstop>
  <tabstop>Injuct_Count_R2</tabstop>
  <tabstop>Set_Injuct_W2</tabstop>
  <tabstop>Injuct_Count_W1</tabstop>
  <tabstop>Injuct_Count_M</tabstop>
  <tabstop>Injuct_Count_R1</tabstop>
  <tabstop>Set_Injuct_R1</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
